packages/scrapers-v2/src/classify/classify.ts

```diff
@@ -2,37 +2,97 @@ import { ensureLengthAtLeast, loadHTML, parseText } from "../utils";
 import { CatalogEntryType, TypedCatalogEntry } from "./types";
 
 export const addTypeToUrl = async (url: URL): Promise<TypedCatalogEntry> => {
-  const type = getUrlType(await loadHTML(url.href));
+  const type = getUrlType(url, await loadHTML(url.href));
   return { url, type };
 };
 
-const getUrlType = ($: CheerioStatic): CatalogEntryType => {
-  const tabsContainer = $("#contentarea #tabs");
+const getUrlType = (url: URL, $: CheerioStatic) => {
```

Comment:
haha good catch, it was because i was passing around a statslogger everywhere, so i needed the URL to recordError, but its not needed anymore (i can remove)


ideally there would be an easier way to setup a global logger so that passing it by param explicitly is not needed. and also a way to do assertions. i thought about it for a bit and decided it wasn't a v1 feature